### DISCUSSION TITLE
MacOS: rar path can't be set

### DISCUSSION THREAD

**Discussion by Gust-Front** [comictagger/comictagger#561] [Q&A]
Putting this in Discussions since I suspect I've missed something obvious.

When I set the rar path in Preferences/RAR Tools, the path appears as light grey and isn't saved. 

<img width="850" alt="Screen Shot 2023-10-30 at 7 57 48 am" src="https://github.com/comictagger/comictagger/assets/10264737/a7577118-1c56-481f-aaff-2f8d2e094c11">

When I try to open a .cbr, I get the 'RAR Files are Read-Only' prompt (obviously, since the path isn't saving.).

This occurs with 1.6.0-alpha.7 & 1.6.0-alpha.8.

With 1.5.5, ComicTagger will open the .cbr but won't save the file: giving the error 'The tag operation seemed to fail'

- I have installed rar with Homebrew.
- I have checked that it's the correct path to rar.
- I have cleared the quarantine flag as per issue [#535](https://github.com/comictagger/comictagger/issues/535).
- I have successfully tested my rar install, as per [#535](https://github.com/comictagger/comictagger/issues/535)

Here is the log:
[ComicTagger.log](https://github.com/comictagger/comictagger/files/13199476/ComicTagger.log)

ComicTagger works fine with .cbz files.

What have I missed?

**Comment by lordwelch** [comment]
ComicTagger does try not to save the path when possible. That's not your issue here at least on 1.6.0-alpha.8, the error is indicating that the files are corrupt can you run `rar t /Users/paul/Downloads/Radiant_Black_026.5.cbr`
